### PR TITLE
Feature/wnb 86/initial routes

### PR DESCRIPTION
**Pull request by ikushlianski**
- add basic routes
- add root components for all routes
- add very basic login/logout flow on UI

### CODE REVIEW

File: src/routes/Signup.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+export default class Signup extends React.Component {
+  // TODO add redirection to "/" in case a user is already registered
+  render() {
+    return <div>Registration page</div>;
```
Comment (vitali-sakalou) [inline src/routes/Signup.js]: Probably this component should render previous version of SignUp component (Same as Signin component)?

**Review by vitali-sakalou — COMMENTED** [review]
Signup component should render previous ver of SignUp component?

File: src/routes/Signup.js (hunk 1 above)
Comment (ikushlianski) [reply src/routes/Signup.js]: We don't yet have a registration page component, do we? I created a separate task for this (WNB-124).

Comment (ikushlianski) [reply src/routes/Signup.js]: hmm... I see you implemented registration form in your previous pull, but when merging, I seem to have somehow lost it... I will restore it

Comment (ikushlianski) [reply src/routes/Signup.js]: I probably see why this happened.. You named your component file SignUp but named the function SignIn and exported it as SignIn https://github.com/Wannabook/wannabook-frontend/blob/develop/src/routes/SignUp.js#L90. This is why during conflict resolution I probably lost your registration component entirely

**Comment by ikushlianski** [comment]
> Signup component should render previous ver of SignUp component?

Done

File: src/routes/Signup.js (hunk 1 above)
Comment (vitali-sakalou) [reply src/routes/Signup.js]: Yes! You are right. I have to be more careful)

**Review by vitali-sakalou — APPROVED** [review]

**Review by letutsenik — APPROVED** [review]

**Pull request merged**